packages/components/src/organisms/RatingSummary/RatingDistribution.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { forwardRef, type HTMLAttributes } from 'react'
+import RatingDistributionItem from './RatingDistributionItem'
+
+export interface RatingDistributionProps
+  extends HTMLAttributes<HTMLOListElement> {
+  /**
+   * The rating distribution
+   */
+  distribution: {
+    starsOne: number
+    starsTwo: number
+    starsThree: number
+    starsFour: number
+    starsFive: number
+  }
+  /**
+   * Optional test ID for testing.
+   */
+  testId?: string
+}
+
+export const RatingDistribution = forwardRef<
+  HTMLOListElement,
+  RatingDistributionProps
+>(function ProgressStatus(
```

Comment:
Should it be `RatingDistribution` ? 
```suggestion
>(function RatingDistribution(
```